### PR TITLE
Fix some Rubocop offenses

### PR DESCRIPTION
**Pull request by amire80**

### CODE REVIEW

**Comment by amire80** [comment]
@vikassy any idea why does it fail?

**Comment by vikassy** [comment]
@amire80 Looks like your rubocop branch does not have .travis.yml

**Comment by amire80** [comment]
Rebased.

**Comment by amire80** [comment]
Rebased, closed, reopened, passed :)

Thanks to the quasiomnipotent StackOverflow! http://stackoverflow.com/questions/17606874/trigger-a-travis-ci-rebuild-without-pushing-a-commit

File: .rubocop.yml [hunk 1]
```diff
@@ -1 +1,9 @@
 inherit_from: .rubocop_todo.yml
+
+# The default is 15, which is too tight
+Metrics/AbcSize:
+  Max: 30
+
+# The default is 20, which is too tight
```
Comment (zeljkofilipin) [inline .rubocop.yml]: If the default is 20, why are you explicitly setting it to 20 in line 9?

Comment (amire80) [inline .rubocop.yml]: Silly, wrong comment. It's 10.

**Pull request merged**